core/unify.ml

```diff
@@ -785,6 +830,10 @@ let rec unify' : unify_env -> (datatype * datatype) -> unit =
        end
     | `End, `End -> ()
     | _, _ ->
+        (*
+        let () = Printf.printf "Match fail in unify: \n%s \n and \n%s"
```

Comment:
Is this for debugging purposes? In other places I've seen `Debug.print` being used instead of `Printf.printf`.